client/.eslintrc.json

```diff
@@ -27,6 +27,7 @@
       "@typescript-eslint"
   ],
   "rules": {
+     "react/prop-types": "off",
```

Comment:
Any reason for turning this off?

Reply:
```suggestion
```
Bumping this again - I didn't see a any issue flagged by eslint when removing this